webapp/src/main/webapp/index.htm

```diff
@@ -38,11 +38,18 @@
         	<td width="100px"><div class="logo"></div></td>
         	<td>
             	<div class="bar-round">
-        			<div class="bar">
-        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
-        				<p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. Learn about the available <a href="https://wiki.openmrs.org/x/d4aIBQ">User Interface Modules</a></p>
-        				<p>If you are a developer, you can access the REST API. (See <a href="https://wiki.openmrs.org/x/P4IaAQ">REST documentation for clients</a>)</p>
-        			</div>
+                    <div class="bar">
+                        <h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+                        <p>If you are seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module
+                            is installed.</p>
+                        <p>
```

Comment:
"Click here to access Add On Manager". Is this grammar correct?

Reply:
Updated.
Is "Click here to access **the** Add On Manager" a better sentence construct?

Reply:
Exactly!